cmake/sysbuild/suit.cmake

```
@@ -231,6 +253,14 @@ function(suit_create_package)
       )
     endif()
     suit_copy_artifact_to_output_directory(${image} ${BINARY_DIR}/zephyr/${BINARY_FILE})
+
+    unset(CONFIG_SUIT_RECOVERY)
+    sysbuild_get(CONFIG_SUIT_RECOVERY IMAGE ${image} VAR CONFIG_SUIT_RECOVERY KCONFIG)
+    if (CONFIG_SUIT_RECOVERY)
```

```suggestion
    if(CONFIG_SUIT_RECOVERY)
```
No spaces between if and bracket in cmake

---

Ok fixed